tests/TestCase/Controller/Component/RequestHandlerComponentTest.php

```diff
@@ -336,7 +337,7 @@ public function testViewClassMap()
     public function testIsAjaxParams()
     {
         $this->request->env('HTTP_X_REQUESTED_WITH', 'XMLHttpRequest');
-        $event = new Event('Controller.startup', $this->Controller);
+        $event = new ControllerEvent('Controller.startup', $this->Controller);
```

Comment:
And wouldn't that break BC? Because the ```dispatchEvent``` method uses the event as well. Any ```dispatchEvent``` call using a event name like ```Controller.startup``` would use ```Controller.Controller.startup``` as name instead. We could possibly check if it has been prefixed already, if not we could add it.